test/framework/src/main/java/org/elasticsearch/indices/recovery/AsyncRecoveryTarget.java

```diff
@@ -83,7 +83,6 @@ public void cleanFiles(int totalTranslogOps, long globalCheckpoint, Store.Metada
     @Override
     public void writeFileChunk(StoreFileMetaData fileMetaData, long position, BytesReference content,
                                boolean lastChunk, int totalTranslogOps, ActionListener<Void> listener) {
-        // TODO: remove this clone once we send file chunk async
```

Comment:
I'm confused. Why not do what the comment says?

Reply:
I assumed we are going to use multiple buffers when I added the comment. But we now use a single buffer instead.